concepts/array-analysis/about.md

```diff
@@ -1,6 +1,8 @@
 # About
 
-In JavaScript, instances of an `Array` are regular objects, decorated with lots of [useful methods][array-docs] from the `Array.prototype`. It includes the `length` property, as well as methods for traversing, analyzing and mutating the array. The array's elements are indexed properties of that array.
+In JavaScript, instances of an [`Array`][arrays-concept] are regular objects, decorated with lots of [useful methods][array-docs].
+Many of the built-in functions that analyse the contents of an array, take a function that returns true or false as an argument. Such a function is called [`predicate`][predicate_in_programming]
```

Comment:
Line 3-6 read a bit strangely now. I would recommend to insert the new sentence in line 4 after line 6 instead. I think that's a better fit.

General sidenote: For the about.md file it is ok if they mention content that the student might not have covered yet, e.g. it would be ok to mention `prototype` here. The reason is that about files are meant as extensive reference for later, they don't have to follow the concept tree structure.